src/server/utilities/cspHeader/index.js

```diff
@@ -361,14 +361,27 @@ const directives = {
     ],
   },
   fontSrc: {
-    amp: [
+    ampLive: [
+      'https://gel.files.bbci.co.uk', // Reith fonts
+      'https://ws-downloads.files.bbci.co.uk', // Other WS fonts
+      'news.files.bbci.co.uk', // STY Includes
```

Comment:
shouldn't this have `https://` in, I think we want to restrict our csp to secure origins? We have the upgrade insecure flag set further down to upgrade any insecure origins used.